lib/src/notification.dart

```diff
@@ -88,23 +88,17 @@ class AndroidChannel {
   });
 }
 
-class IOSPermissions {
-  final bool alert;
-  final bool badge;
-  final bool sound;
-  const IOSPermissions({
-    this.alert = true,
-    this.badge = true,
-    this.sound = true,
-  });
```

Comment:
I mentioned it in Slack. Here's the context:

Right now, on iOS, we implicitly request for push notification permissions when a user calls `registerPushNotificationHandlers`. On Android, before version 13, there was no need to request for push notification permission. That is no longer the case. 

So, rather than implement something similar for Android, I went with the route of removing this implicit call and letting the user handle requesting permissions on their own. This way, they can request permissions at any point in their app’s user flow rather than it being almost immediately after the app is opened as was the case before. With this in mind, I removed the permission related properties from iOSPermissions. We shall document what users need to do and how they can request for the necessary permissions on both platforms for React Native and Flutter.

Reply:
OK, thanks for the explaination👍